### DISCUSSION TITLE
School issued email addres

### DISCUSSION THREAD

**Discussion by kangsoekme** [community/community#130680] [GitHub Education]
### Hi everyone,

Hi everyone,

I have a new domain email for my education school until i can't register to github education, how to i add my email domain for i can to register to github education?
![Screenshot 2024-06-29 182540](https://github.com/community/community/assets/134789825/7168e1a6-3a43-4c89-b51e-f933eb8e0938)

**Comment by kangsoekme** [comment]
my domain is student.cs.unida.gontor.ac.id

**Reply by YuvrajxChopra** [reply]
You need to verify your email address. Just put your education email address in your account. Go to https://github.com/settings/profile this page and you'll see the email option on the left sidebar, add your email there and verify it.

**Comment by kangsoekme** [comment]
Dear Github

i had added my school email addres in my github profile.  but I still can't
register to github education. I hope your support until I can register to
Github education .






Your regards
Abdullah Sukma Jati
